content/docs/lists-and-keys.md

```diff
@@ -119,34 +122,35 @@ const todoItems = todos.map((todo) =>
 );
 ```
 
-When you don't have stable IDs for rendered items, you may use the item index as a key as a last resort:
+Ketika anda tidak memiliki ID yang stabil untuk me-*render item*, anda bisa menggunakan index dari *item* sebagai *key* untuk pilihan terakhir:
 
 ```js{2,3}
 const todoItems = todos.map((todo, index) =>
   // Only do this if items have no stable IDs
```

Comment:
I think this should be removed, since it has been translated below

Reply:
owh.. sorry didn't notice about that paragraph